src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -8215,31 +8268,65 @@ void checkWellKnown(WellKnownMember member)
                             original.Parameters[0] is { Type: { SpecialType: SpecialType.System_Int32 }, RefKind: RefKind.None } &&
                             original.Parameters[1] is { Type: { SpecialType: SpecialType.System_Int32 }, RefKind: RefKind.None })
                         {
-                            patternSymbol = method;
+                            method.AddUseSiteInfo(ref useSiteInfo);
+                            diagnostics.ReportUseSite(method, syntax);
```

Comment:
>diagnostics.ReportUseSite(method, syntax); [](http://example.com/codeflow?start=28&length=42)

Since we are doing this, the previous line looks unnecessary. Also, I think the process of binding the call is supposed to report use-site info for the candidate. So, is this operation redundant?